pennylane/fermi/fermionic.py

```diff
@@ -108,8 +108,67 @@ def __repr__(self):
         r"""Terminal representation of a FermiWord"""
         return str(self)
 
+    def __add__(self, other):
+        """Add a FermiSentence, FermiWord or constant to a FermiWord. Converts both
+        elements to be added into FermiSentences, and uses the FermiSentence __add__
+        method"""
+
+        self_fs = FermiSentence({self: 1.0})
+
+        if isinstance(other, FermiSentence):
+            return self_fs + other
+
+        if isinstance(other, FermiWord):
+            other_fs = FermiSentence({other: 1.0})
+            return self_fs + other_fs
+
+        if isinstance(other, (float, int, complex)):
+            other_fs = FermiSentence({FermiWord({}): other})
+            return self_fs + other_fs
+
+        raise TypeError(f"Cannot add {type(other)} to a FermiWord.")
+
+    def __radd__(self, other):
+        """Add a FermiWord to a constant, i.e. `2 + FermiWord({...})`"""
+
+        if isinstance(other, (float, int, complex)):
+            return self.__add__(other)
+
+        raise TypeError(f"Cannot add a FermiWord to {type(other)}.")
+
+    def __sub__(self, other):
+        """Subtract a FermiSentence, FermiWord or constant from a FermiWord. Converts both
+        elements to be added into FermiSentences (with negative coefficient for `other`), and
+        uses the FermiSentence __add__  method"""
+
+        self_fs = FermiSentence({self: 1.0})
+
+        if isinstance(other, FermiWord):
+            other_fs = FermiSentence({other: -1.0})
+            return self_fs + other_fs
+
+        if isinstance(other, FermiSentence):
+            other_fs = FermiSentence(dict(zip(other.keys(), [-v for v in other.values()])))
+            return self_fs + other_fs
+
+        if isinstance(other, (float, int, complex)):
+            other_fs = FermiSentence({FermiWord({}): -1 * other})  # -constant * I
+            return self_fs + other_fs
+
+        raise TypeError(f"Cannot subtract {type(other)} from a FermiWord.")
+
+    def __rsub__(self, other):
+        """Subtract a FermiWord to a constant, i.e. `2 - FermiWord({...})`"""
```

Comment:
How about subtracting a `FermiSentence`? That is channelled by the `FermiSentence` `__sub__ `method?

Reply:
yes :)